docs/database/updates/2016-12-22_1-schema.sql

```diff
@@ -0,0 +1,25 @@
+CREATE TABLE `audios` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `sentence_id` int(11) NOT NULL,
+  `user_id` int(11) DEFAULT NULL,
+  `external` varchar(500) DEFAULT NULL,
+  `created` datetime NOT NULL,
+  `modified` datetime NOT NULL,
+  PRIMARY KEY (`id`),
+  KEY `sentence_id` (`sentence_id`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+ALTER TABLE users
+  ADD `audio_license` varchar(50) CHARACTER SET utf8 COLLATE utf8_unicode_ci DEFAULT NULL,
+  ADD `audio_attribution_url` varchar(255) CHARACTER SET utf8 COLLATE utf8_unicode_ci DEFAULT NULL;
+
+INSERT INTO audios
+(sentence_id, created, modified)
+SELECT
+  id as sentence_id,
+  0 as created,
+  0 as modified
+FROM sentences
+WHERE hasaudio != 'no';
+
+ALTER TABLE sentences DROP hasaudio;
```

Comment:
I've been thinking maybe we shouldn't drop the `hasaudio` field yet. We should make it deprecated, and keep it there for long enough so that:
- we have time to re-adapt the deduplication script
- we have time to make sure there's nothing else of our scripts/tools that will break

This would mean of course re-adapting the implementation in this PR so that `hasaudio` still gets updated accordingly when adding/removing audio.